src/panels/welcome/welcome.js

```diff
@@ -17,14 +18,7 @@ export function initializePanel(panelType, data) {
   }
   
   // Event listener for analyze button
-  analyzeBtn.addEventListener('click', () => {
-    // For now, just show a simple message
-    analyzeBtn.disabled = true;
-    analyzeBtn.textContent = 'Coming Soon!';
-    
-    setTimeout(() => {
-      analyzeBtn.disabled = false;
-      analyzeBtn.textContent = 'Analyze this website';
-    }, 2000);
+  analyzeBtn.addEventListener('click', async () => {
+    await extensionManager.openPanel('results');
```

Comment:
Noting that the button from the welcome panel (and all of the other welcome panel elements) are still in the DOM when I'm on the results panel because the welcome panel is still open, would we want to remove those? Maybe over here when we open the results panel, also close the panel? (I think that may visually animate the welcome panel to slide out then slide in the results panel though, is that undesirable?)

<img width="320" height="920" alt="Screenshot 2025-10-21 at 12 16 13 PM" src="https://github.com/user-attachments/assets/00876641-98e8-4ce8-be24-bd771cc79d75" />
<img width="366" height="282" alt="image" src="https://github.com/user-attachments/assets/5d91ccc4-d399-422d-9bbf-b11df3dcf52a" />